src/placeos-models/user_jwt.cr

```diff
@@ -62,6 +65,72 @@ module PlaceOS::Model
       @id
     end
 
+    def scope_public?
+      scope.includes?(PUBLIC)
+    end
+
+    def scope_guest?
+      scope.includes?(GUEST)
+    end
```

Comment:
We may have more cases, like `public:none`/`guest:read`

Reply:
That is checked in rest-api

https://github.com/PlaceOS/rest-api/blob/ouath-scope/src/placeos-rest-api/utilities/current-user.cr#L93-L97

Reply:
I think having these like `guest_scope?` or event `guest?` might read better